### ISSUE TITLE
Add checks to avoid duplicated YAML frontmatter keys

### ISSUE THREAD

**Issue by jrgant** [jrgant/ox-quarto#1]
Right now, the exporter doesn't check to see if keys extracted from the org file duplicate keys provide via `QUARTO_OPTIONS` or `QUARTO_FRONTMATTER`

**Comment by jrgant** [comment]
I would be reasonable to let `quarto-cli` handle the checking, as it already does.

**Comment by fleimgruber** [comment]
This is a blocker for https://github.com/jrgant/ox-quarto/issues/14 where I am working on the example file and an `example.yaml` frontmatter:

```yaml
author:
  - name: ox-quarto contributors
```

With any org file that does not have `#+AUTHOR:` set errors from quarto-cli with:

```
Process quarto-render exited abnormally with code 1
ERROR: YAMLException: duplicated mapping key (ox-quarto-example.qmd, 6:1)
5: author:
  ~~
6:   - name: ox-quarto contributors

Process quarto-render exited abnormally with code 1
```

I assume because Org mode inserts a default author value if none is given explicitly and thus it errors the same way as if `#+AUTHOR:` was given (i.e. what this issue describes)?

**Comment by jrgant** [comment]
Yes, I think that's what's happening.

Best way to fix is probably to leave this to org: `#+OPTIONS: author:nil`.

**Comment by fleimgruber** [comment]
> Best way to fix is probably to leave this to org: `#+OPTIONS: author:nil`.

Yes, that is also an option. #16 (without an `author` in frontmatter) works with the default Org author value.
